moped-database/migrations/1704919187107_moped_proj_funding_rename_columns/up.sql

```diff
@@ -0,0 +1,34 @@
+ALTER TABLE moped_proj_funding RENAME COLUMN date_added TO created_at;
+ALTER TABLE moped_proj_funding RENAME COLUMN added_by TO created_by_user_id;
+
+ALTER TABLE moped_proj_funding ADD COLUMN updated_at TIMESTAMP WITH TIME ZONE;
+ALTER TABLE moped_proj_funding ADD COLUMN updated_by_user_id INTEGER;
+
+COMMENT ON COLUMN moped_proj_funding.updated_at IS 'Timestamp when the record was last updated';
+COMMENT ON COLUMN moped_proj_funding.updated_by_user_id IS 'ID of the user who last updated the record';
+
+CREATE OR REPLACE FUNCTION public.update_self_and_project_updated_audit_fields() RETURNS TRIGGER
+    LANGUAGE plpgsql
+    AS $$
+BEGIN
+    -- Update the updated_at field in the current row of the triggering table
+    NEW.updated_at := NOW();
+
+    -- Update the updated_at and updated_by_user_id fields in the related row of moped_project
+    UPDATE moped_project
+    SET updated_at = NOW(),
```

Comment:
looking at this again @frankhereford, I think we probably want to to use the value from `NEW.updated_at` rather than calling `NOW()` again here? just to ensure those two values are 100% identical?

Reply:
That sounds great John, I'm in there right now addressing some feedback from Mike, and I'll get that in too.

Reply:
Great idea - using the first timestamp in the second operation will ensure they are always identical. It's in there.